packages/datadog-plugin-aws-sdk/src/index.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const Tags = require('opentracing').Tags
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const awsHelpers = require('./helpers')
+
+function createWrapRequest (tracer, config) {
+  config = normalizeConfig(config)
+
+  return function wrapRequest (send) {
+    return function requestWithTrace (cb) {
+      if (!this.service) return send.apply(this, arguments)
+
+      const serviceName = this.service.serviceIdentifier
+      const childOf = tracer.scope().active()
+      const baseTags = {
+        [Tags.SPAN_KIND]: 'client',
+        'span.type': 'http',
```

Comment:
This should be `aws` if we support it in the backend, and otherwise it shouldn't be set.

Reply:
ok, removed. I don't think `aws` is supported, so removing this. Fwiw this was just copying the serverless convention, which does assign `dynamodb` it's own span.type, so i've left that bit in within the `DynamoDbService` class